webpack/assets/javascripts/react_app/components/BreadcrumbBar/__tests__/__snapshots__/BreadcrumbBar.test.js.snap

```diff
@@ -110,7 +110,7 @@ Array [
       "switcherItemUrl": "some/url/:id",
     },
     Object {
-      "page": 1,
```

Comment:
Just a question, not blocking: is page differences expected? If so, why?

Reply:
```
    rerender(
        <BreadcrumbBar
          {...{ ...props, isSwitcherOpen: true, currentPage: 2, total: 40 }}
        />
      );
``` 
 -> page is 2
```
      await act(async () =>
        fireEvent.click(screen.getByLabelText('Go to next page'))
      );
      expect(props.loadSwitcherResourcesByResource.mock.calls).toHaveLength(2);
```
-> page is 3 in the second call

```
   await act(async () =>
        fireEvent.click(screen.getByLabelText('Go to previous page'))
      );
      expect(props.loadSwitcherResourcesByResource.mock.calls).toHaveLength(3);
```

-> page is 2 in the 3rd call

Reply:
Oh, my bad, should've looked at the test better :/ Thank you :)